sdk/core/core-util/src/delay.ts

```diff
@@ -2,9 +2,8 @@
 // Licensed under the MIT license.
 
 import { AbortError, AbortSignalLike } from "@azure/abort-controller";
-import { isDefined } from "./typeGuards";
 
-const StandardAbortMessage = "The operation was aborted.";
+const StandardAbortMessage = "The delay was aborted.";
```

Comment:
I think some Storage tests at least might be sniffing the message, so you could see some failures with brittle tests. I tried to clean most of that up recently, but fair warning. :)

Reply:
The packages in the CI pipeline seem to be happy but if you come across any such failure, please pass it my way and I will address it.